### ISSUE TITLE
[HOLD #11189] Chat - Unable to upload all file types

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#12459]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___
Bug found when executing #11313
## Action Performed:
1. Go to URL https://staging.new.expensify.com/
2. Log in with any account
3. Go to any chat
4. Try to uploading .mkv, .psd, 3gp and .flv files

## Expected Result:
Files should get uploaded

## Actual Result:
Error appears when uploading the files

## Workaround:
unknown

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.2.23-7
**Reproducible in staging?:** y
**Reproducible in production?:** y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 
**Expensify/Expensify Issue URL:** 
**Issue reported by:** Applause - internal team
**Slack conversation:** 

https://user-images.githubusercontent.com/43996225/199866985-2d0ad5db-4416-4d3a-9f70-acb076566d48.mp4

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by sakluger** [comment]
@youssef-lr is assigning this to himself because it's related to https://github.com/Expensify/App/issues/11189.

**Comment by youssef-lr** [comment]
This has been reverted due to the bug in this issue, I will put this on HOLD as I'm working on bringing back support for all filetypes.

**Comment by JmillsExpensify** [comment]
Re-assigned myself per [this comment](https://expensify.slack.com/archives/C01GTK53T8Q/p1667850709001799?thread_ts=1667848459.557979&cid=C01GTK53T8Q).

**Comment by JmillsExpensify** [comment]
Moved to weekly based on the hold.

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by youssef-lr** [comment]
On hold.

**Comment by JmillsExpensify** [comment]
Yeah, this is less of a priority compared to refactors. Let's keep the focus there for now.

**Comment by JmillsExpensify** [comment]
Still not a priority over the course of WAQ.

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (Second week)

**Comment by JmillsExpensify** [comment]
@youssef-lr should we still keep this issue open still waiting on #11189? Given that Applause isn't able to reproduce now, and then the linked issue is on hold for WAQ, I'm inclined to close this one.

**Comment by youssef-lr** [comment]
I agree we can close this @JmillsExpensify.

**Comment by JmillsExpensify** [comment]
Great thanks!
